### ISSUE TITLE
Badge amound != Ledger Amount

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#1839]
In my [project](https://github.com/amihaiemil/docker-java-api), the Cash balance displayed in the Project's homepage/Ledger differs from the one displayed on the ZC badge from README.md. 

The one from the Ledger is the correct one. Maybe the one displayed on the Badge also deducts 0crat's fees? But the project is opensource, its fees should be waived.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) not enough funds available in the project, can't set budget of job [#1839](https://github.com/zerocracy/farm/issues/1839), see [§21](http://www.zerocracy.com/policy.html#21); @g4s8[/z](https://www.0crat.com/u/g4s8) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by amihaiemil** [comment]
@g4s8 hmm, so everything is alright, after all?

**Comment by g4s8** [comment]
@amihaiemil no, balance value is not correct (origin issue https://github.com/zerocracy/farm/issues/1839#issue-395739135), but locked amount is OK if you have non-zero rates in the project https://github.com/zerocracy/farm/issues/1839#issuecomment-451407551
